### ISSUE TITLE
Can't change path to hard file

### ISSUE THREAD

**Issue by randalemicha** [lubomyr/uae4arm#6]
I'm running uae4arm on Shield TV. My hardfiles are located on a network drive (/storage/network_drive). uae4arm does not locate to this path. It only accepts paths under /storage/emulated/0/... 
I also tried changing the path manually in configuration, but without success.

**Comment by lubomyr** [comment]
How did you change manually path ? Did you edited adfdir.conf ?

27 трав. 2017 11:14 "randalemicha" <notifications@github.com> пише:

**Comment by randalemicha** [comment]
I edited adfdir.conf and also tried to modify the .uae file directly. When I edit "path" in adfdir.conf and try to add a hardfile it shows the correct path, but as soon as I click "..." to choose a file I'm trapped in /storage/emulated/0 again.

**Comment by miker1117** [comment]
It works for me, when you edit the conf, make it the root of the area your trying to access, or just put the entire address including the .hdf on the conf, I have 2 hdds set up, 9ne called phone the other SDCard, works fine...

BUT I get a write protect error on any write bto the SDCard, Directory or .HDF

**Comment by Sobriquet42** [comment]
> It works for me, when you edit the conf, make it the root of the area your trying to access, or just put the entire address including the .hdf on the 

Confirmed, you have to create a .uae file with the exact path (case sensitive) to the hdf file you want to mount, it works in read only as network shares are stuck to read only on the Shield. 

Changing the home directory to the netwoirk share does not work, the browser resets so /storage/emulated/0/ whenever you select something.

It is really inconvenient when dealing with large collections of hdf roms to have to handcraft every single rom its own .uae config file. Would appreciate a fix on the browser.

**Comment by miker1117** [comment]
> > It works for me, when you edit the conf, make it the root of the area your trying to access, or just put the entire address including the .hdf on the conf
> 
> Confirmed, you have to create a .uae file with the exact path (case sensitive) to the hdf file you want to mount, it works in read only as network shares are stuck to read only on the Shield.
> 
> Changing the home directory to the netwoirk share does not work, the browser resets so /storage/emulated/0/ whenever you select something.
> 
> It is really inconvenient when dealing with large collections of hdf roms to have to handcraft every single rom its own .uae config file. Would appreciate a fix on the browser.

Agreed,  I’ve basically been having this issue since Android 5?  And above, normally when I’m able my phone is rooted which basically gets rid of googles (or the isp) lick downs....

What I can’t figure is why I can’t re-permission my Sd card... I mean I bought it I inserted it I should be able to say WHAT MY PERMISSION level is....

Mike
